cartridges/int_cloudinary_headless/cartridge/scripts/hooks/productDetails.js

```diff
@@ -10,16 +10,48 @@ exports.modifyGETResponse = function (product, doc) {
     try {
         if (cloudinaryConstants.CLD_ENABLED) {
             var cldPageSetting = cloudinaryConstants.CLD_IMAGE_PAGE_TYPE_SETTINGS_OBJECT;
-            var cloudinary = {};
+
+            // Cloudinary plp images for recommendation tile
+            var cloudinary = cloudinaryModel.getProductPrimaryImage(product.ID,
```

Comment:
@asad-rafter the cloudinary var seems to hold the PLP image, so please rename accordingly.
In general, a variable named 'cloudinary' cannot be a meaningful name in this cartridge